client_configuration_test.go

```diff
@@ -71,13 +72,15 @@ func TestNotFindConfigFileWhenNotDefined(t *testing.T) {
 }
 
 func TestCreatePredefinedDirs(t *testing.T) {
+	exeDir, _ := os.Executable()
```

Comment:
if this is a symlink should we look for config in the place where symlink is located or evaluate the symlink and look where it points to?